### PR TITLE
[no-Jira] Remove storybook and chromatic

### PR DESCRIPTION
**Pull request by canac**
## Description

Remove storybook and chromatic. We are not using them, and keeping them adds a maintenance burden of upgrading to the latest version and updating the story files.

This will also resolve some dependabot warnings from webpack v4.

## Checklist:

- [x] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [x] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [x] I have requested a review from another person on the project

### CODE REVIEW

File: pages/accountLists/[accountListId]/contacts/[[...contactId]].stories.tsx
Comment (dr-bizz) [inline pages/accountLists/[accountListId]/contacts/[[...contactId]].stories.tsx:1]: Were you meant to get rid of this component? I don't think it's used

**Review by dr-bizz — APPROVED** [review]
Thank you for removing storybook

**Comment by dr-bizz** [comment]
Let's merge it with staging or put it on a feature branch before merging it into `main`, just in case. I don't need to test it, if you just check it, that is fine.

File: pages/accountLists/[accountListId]/contacts/[[...contactId]].stories.tsx
Comment (canac) [reply pages/accountLists/[accountListId]/contacts/[[...contactId]].stories.tsx:1]: I think I did. There are no `*.stories.*` files in the repo anymore.

**Pull request merged**